packages/meditrak-server/src/tests/routes/surveyResponse.test.js

```diff
@@ -394,17 +395,17 @@ describe('surveyResponse endpoint', () => {
 
     const { surveyResponseId } = body.results[0];
     const dbResponse = await models.surveyResponse.findOne({ id: surveyResponseId });
-    expect(momentTimezone.tz(dbResponse.submission_time, dbResponse.timezone).format()).to.be.oneOf(
-      [
-        '2019-07-31T06:48:00+13:00',
-        // While 'Antarctica/McMurdo' is in DST time, it will be selected as the timezone,
-        // however the timestamp is hardcoded to a date outside of DST time for McMurdo, so it will be
-        // parsed with it's non-DST +12 offset.
-        '2019-07-31T05:48:00+12:00',
-      ],
-    );
+    expect(moment.utc(dbResponse.data_time).tz(dbResponse.timezone).format()).to.be.oneOf([
+      '2019-07-31T06:48:00+13:00',
+      // While 'Antarctica/McMurdo' is in DST time, it will be selected as the timezone,
+      // however the timestamp is hardcoded to a date outside of DST time for McMurdo, so it will be
+      // parsed with it's non-DST +12 offset.
+      '2019-07-31T05:48:00+12:00',
+    ]);
```

Comment:
I find this a weird test case. I know you're trying to make the best of the old case, but I just don't think it works as well, or is even necessary, with `data_time`. Let's just remove it, unless you want to argue for its value?